languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -11774,7 +11774,47 @@ USA
             </rule>
         </rulegroup>
     </category>
+
+
     <category id='REMOVE_PRONOUN' name='Remoção de pronomes' type='style' tone_tags="povrem">
+
+
+        <!-- POR ELE SABER por saber -->
+        <rule id='NOMECOMUM_PRONOME_VERBINF_REM_PRON' name="Simplificar: Nome Comum + Pronome + Verbo Infinitivo → Omitir Pronome" type='style' tone_tags="povrem">
```

Comment:
Not sure if you've seen it, but I've already been replacing all instances of 'nome' with 'substantivo'. The former isn't really used to mean 'noun' in pt-BR, so using 'substantivo' should be more inclusive. In fact, I'm not entirely sure myself why the rule ID makes reference to common nouns, since the regexp used to define the noun token specifically allows for `NP`.